src/index.js

```diff
@@ -29,5 +33,22 @@ ReactDOM.render(
       </Router>
     </NodeProvider>
   </Provider>,
-  document.getElementById('root')
+  root
 );
+
+injectGlobal`
+  html,
+  body,
+  input,
+  button,
+  select,
+  option {
+    color: ${COLORS.gray[900]};
+    font-family: 'Futura PT';
+  }
+
+  body {
+    background: ${COLORS.gray[50]};
+    color: ${COLORS.gray[900]};
```

Comment:
Ah, my bad; I realized that we don't actually need to duplicate `color` since we added `body` to the first group!

No matter, though. I'll tidy up quickly after merging :)